search-api/src/search_api/utils/util.py

```diff
@@ -29,7 +29,7 @@ def options(self, *args, **kwargs):  # pylint: disable=unused-argument
             return {'Allow': 'GET'}, 200, \
                    {'Access-Control-Allow-Origin': '*',
                     'Access-Control-Allow-Methods': methods,
-                    'Access-Control-Allow-Headers': 'Authorization, Content-Type'}
+                    'Access-Control-Allow-Headers': 'Authorization, Content-Type, App-Name'}
```

Comment:
Thanks for this fix.

So this should work now?

![image](https://github.com/user-attachments/assets/f45fd855-cf89-420c-87b6-daa1ee8d6132)

I still get this:

![image](https://github.com/user-attachments/assets/4e04fb8e-2948-45c4-9318-ad830ae0c401)

Reply:
@kialj876 This is still not working, as discussed.

No rush on a fix. I will instead update the UIs to not add App-Name for reg search calls.

Later, if desired for logging, we can add the App-Name header back in for reg search calls.